### DISCUSSION TITLE
What does "Save item dates before proceeding" mean?

### DISCUSSION THREAD

**Discussion by rayyangh** [deckerst/aves#1147] [Q&A]
appears when I move an image from one album to another.
![SmartSelect_20240823_151139_Aves Libre](https://github.com/user-attachments/assets/c486457e-0e83-432f-8cef-12918414f471)

Thank you in advance for your time and help!

**Comment by deckerst** [comment]
This saves the current "file modified date" as a "metadata date", and it is suggested when the file being moved (or modified) does not already have a metadata date. 

The reason the app suggests this is because metadata dates are more reliable than file attributes. Aves is a gallery app and not a file manager app, so it cannot control file attributes and the "file modified date" won't be preserved when moving files.

**Reply by sshaikh** [reply]
That's a handy feature.

However my confusion with this was seeing the message when deleting files. Is it possible to suppress it in that particular situation?

**Reply by deckerst** [reply]
Deleting is actually a move action (to the bin), so if you restore the item you'll be glad the date was saved. In any case, it's best to enable it and check "do not ask again".

**Reply by sshaikh** [reply]
Doh of course. I can't imagine not requiring it. Thanks.

**Reply by qyurila** [reply]
Sorry for hijacking a post but quick question, is there any way to change the behavior after choosing 'Do not ask again' once? I think there should be a setting for this but can't find it.

**Reply by deckerst** [reply]
@qyurila you can check these in the Settings > Navigation > Confirmation dialog.

**Comment by deckerst** [comment]
Related FAQ: [What's in a date?](https://github.com/deckerst/aves/wiki/FAQ#whats-in-a-date)

**Reply by SilverGreen93** [reply]
Hi @deckerst 

Just encountered this message today and had no idea what it wants from me.

I would suggest to rename the text to: "Write file modified date into photo metadata?" (or similar)

It is way more clear to be understood. What do you think?

**Reply by deckerst** [reply]
True, the wording could be improved but 1) users who don't understand this dialog often don't know much about metadata in general, and 2) I don't want to bother 30+ translators for this minor change.

**Reply by sshaikh** [reply]
Is the term "exif" largely foreign (I dont know)?

Or how about a link to the FAQ? In the dialogue box?

**Reply by deckerst** [reply]
> Is the term "exif" largely foreign (I dont know)?
> 
> Or how about a link to the FAQ? In the dialogue box?

I like the idea of linking to the FAQ. I'll think about it, thanks.

**Reply by SilverGreen93** [reply]
> don't want to bother 30+ translators for this minor change.

I think this argument is not valid. If something has to change for better usability/readability from UX perspective, it has to change. Period. Today translations can be made even by AI if a person can't be bothered to update it.

I know about metadata and file dates and still had no idea about what it wanted from me. People using photos and custom apps mostly know what metadata is, otherwise will never use Aves anyway ;)

**Reply by qyurila** [reply]
I agree that this message should be rephrased, mainly because choosing the wrong option here causes an irreversible change (at least seemingly) in the order the images are shown. It would be good if one of the options were marked as 'recommended' that most users would be happy with at least.
